display:
  displayNum: 0
  height: 768
  screenNum: 0
  virtualDisplays:
  - id: vdisp
    viewport:
    - [0, 0]
    - [1024, 0]
    - [1024, 0]
    - [0, 768]
  width: 1024
  windowDecoration: true
  x: 0
  y: 0
geom:
  axis: {x: 0.0, y: 0.0, z: 0.144}
  base: {x: 0.0, y: 0.0, z: 0.0}
  model: cylinder
  radius: 0.046
p2c: /home/strawlab/vros-devel/vros/vros_display/config/p2c.exr
p2g: /home/strawlab/vros-devel/vros/vros_display/config/p2g.exr
stimulus_plugins: []
